livekit-android-sdk/src/main/java/io/livekit/android/LiveKit.kt

```diff
@@ -132,7 +135,10 @@ class LiveKit {
             val protocolVersion = maxOf(options.protocolVersion, ProtocolVersion.v7)
             val connectOptions = options.copy(protocolVersion = protocolVersion)
 
-            room.connect(url, token, connectOptions)
```

Comment:
This should be added to the room.connect function itself rather than this one. `LiveKit.connect` is deprecated and will be removed in the future.